.travis.yml

```diff
@@ -16,14 +16,19 @@ script:
   - npm run eslint
   - npm run test-cov
 
+before_deploy:
```

Comment:
For clearer view, suggest to move `before_deploy` between `after_script` and `deploy`. I understand it doesn't change [the order](https://docs.travis-ci.com/user/job-lifecycle/) (i.e. after_script is always executed last).

Reply:
Thank you for the review.
The timing of `before_deploy` was `npm run jsdoc` because it was involved in deployment, so I wanted to make it more declarative, so I decided this timing.

Reply:
I misunderstood, I was asking you to sort each timing, then I will do it today :)

Reply:
yeah, what I meant was reorder the lines while keeping `after_script`.

`after_success` does make more sense, as coveralls should run before deployment or immediately after `script`.